tests/system/src/test/java/test/robot/javafx/scene/TransparentLCDTest.java

```diff
@@ -0,0 +1,272 @@
+    private static final Color OPAQUE_COLOR = makeOpaque(TRANSP_COLOR);
+
+    private Robot robot;
+    private Stage testStage;
+    private Scene testScene;
+
+    /**
+     * Derive an opaque color from the given transparent color. The opaque
+     * color will be the same color as the transparent color is when rendered
+     * on a pure white background.
+     *
+     * @param c the transparent color
+     * @return  the equivalent opaque color
+     */
+    private static Color makeOpaque(Color c) {
+        double a = c.getOpacity();
+        double r = c.getRed() * a + (1.0 - a);
+        double g = c.getGreen() * a + (1.0 - a);
+        double b = c.getBlue() * a + (1.0 - a);
+        return Color.color(r, g, b);
+    }
+
+    private boolean isGrayScale(List<Color> colors) {
+        long nonGrayCount = colors.stream()
+                .filter(c -> c.getRed() != c.getGreen() || c.getRed() != c.getBlue())
+                .count();
+        return nonGrayCount == 0;
+    }
+
+   private void assertColorEquals(Color expected, Color actual) {
+        if (!testColorEquals(expected, actual, TOLERANCE)) {
+            fail("expected:" + expected + " but was:" + actual);
+        }
+    }
+
+    private boolean testColorEquals(Color expected, Color actual, double delta) {
+        double deltaRed = Math.abs(expected.getRed() - actual.getRed());
+        double deltaGreen = Math.abs(expected.getGreen() - actual.getGreen());
+        double deltaBlue = Math.abs(expected.getBlue() - actual.getBlue());
+        double deltaOpacity = Math.abs(expected.getOpacity() - actual.getOpacity());
+        return (deltaRed <= delta && deltaGreen <= delta && deltaBlue <= delta && deltaOpacity <= delta);
+    }
+
+    private List<Color> getColors(Scene scene, int x, int y, int width) {
+        x += scene.getX() + scene.getWindow().getX();
+        y += scene.getY() + scene.getWindow().getY();
+        Image image = robot.getScreenCapture(null, x, y, width, 1);
+        List<Color> colors = new ArrayList<>(width);
+        for (int i = 0; i < width; i++) {
+            colors.add(image.getPixelReader().getColor(i, 0));
+        }
+        return colors;
+    }
+
+    // This must be called on the FX app thread
+    private Stage createStage() {
+        Stage stage = new Stage();
+        stage.initStyle(StageStyle.UNDECORATED);
+        stage.setAlwaysOnTop(true);
+        return stage;
+    }
+
+    @BeforeAll
+    public static void doSetupOnce() {
+        System.setProperty("prism.lcdText", "true");
+        System.setProperty("glass.win.uiScale", "1");
+        System.setProperty("glass.gtk.uiScale", "1");
+
+        Platform.setImplicitExit(false);
+        final CountDownLatch launchLatch = new CountDownLatch(1);
+        Util.startup(launchLatch, launchLatch::countDown);
+        assertEquals(0, launchLatch.getCount());
+    }
+
+    @AfterAll
+    public static void doTeardownOnce() {
+        Util.shutdown();
```

Comment:
I am sorry, Kevin, you just walked into a trap carefully laid by me before.
Util.shutdown() needs a Stage argument, otherwise it does nothing.
Perhaps we could change the impl. to shutdown all the stages regardless instead of passing an array of them.

And, of course, there is more than one place where we do it.  Good job, me!

Reply:
We can leave this code as is and fix it during the test sprint...  I'll file a bug.

Reply:
One could argue that the method is fine as it is. It will first hide the stages that are passed in (possibly zero of them), and then shutdown the platform. I guess I don't see the problem.

Note that by the time this method is called, there are no visible stages.

Reply:
I guess the reason for explicit hiding() of the stages passed to it was done for the sake of headless tests?

Reply:
It's not really a headless vs headful distinction (the system tests are all headful). It's just that some tests have a global (static) Stage that should be hidden at shutdown time. Others, like the test in this PR, create, hide, and show a Stage in each test instance. Both are valid use cases.

Reply:
here it's under @AfterAll, not after each test.

But still, what's the point of hiding the stage(s) if we follow it with Platform.exit() ?
Then we should get rid of the stage(s) argument and simply hide all the current stages, regardless.

Reply:
For this test, the call to shutdown is in an `@AfterAll` method. The call to `Stage.hide` is in an `@AfterEach` method.